docs/CHANGELOG.asciidoc

```diff
@@ -72,6 +72,417 @@ Review important information about the {kib} 7.16.x releases.
 
 coming::[7.16.0]
 
+For information about the {kib} 7.16.0 release, review the following information.
+
+[float]
+[[breaking-changes-7.16.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and performance.
+Before you upgrade to 7.16.0, review the breaking changes, then mitigate the impact to your application.
+
+// tag::notable-breaking-changes[]
+      
+[discrete]
+[[breaking-112523]]
+.Updates browser support
+[%collapsible]
+====
+*Details* +
+Google Chrome, Microsoft Edge, Mozilla Firefox, and ApplemSafari browser support has been updated. For more information, refer to {kibana-pull}112523[#112523].
+
+*Impact* +
+Use the following supported browsers:
+
+* Chrome 92
+* Edge 92
+* Firefox 91.0
+* Safari 14.1
+====
+      
+[discrete]
+[[breaking-111339]]
+.Removes legacy data table
+[%collapsible]
+====
+*Details* +
+The legacy aggregation-based data table and `vis_type_table.legacyVisEnabled` setting are no longer supported. For more information, refer to {kibana-pull}111339[#111339].
+
+*Impact* +
+To auto-fit the table row height, create an <<<aggregation-reference,aggregations>>, click *Options*, then select *Auto fit rows to content*.
```

Comment:
It's possible that this will fix it?

```suggestion
To auto-fit the table row height, create an <<aggregation-reference,aggregations>>, click *Options*, then select *Auto fit rows to content*.
```
I don't know why, locally, `docbldkb` runs fine without erroring.